tests/Microsoft.Bot.Builder.Dialogs.Tests/TestUtilities.cs

```diff
@@ -13,6 +13,22 @@ namespace Microsoft.Bot.Builder.Dialogs.Tests
 {
     public class TestUtilities
     {
+        private static Lazy<Dictionary<string, string>> environmentKeys = new Lazy<Dictionary<string, string>>(() =>
```

Comment:
Why was the private access modifier added? Was it intentional?

Reply:
Hi @axelsrz, the `private` access modifier was added as it's required by a StyleCop rule. Here is a screenshot which shows the warning that Stylecop raises when there is none access modifier:

![image](https://user-images.githubusercontent.com/43762887/54779039-fb326c80-4bf4-11e9-85c1-af41c8c68a43.png)